supertokens_python/recipe/session/cookie_and_header.py

```diff
@@ -140,12 +141,13 @@ def set_cookie_response_mutator(
     value: str,
     expires: int,
     path_type: Literal["refresh_token_path", "access_token_path"],
+    user_context: Dict[str, Any],
```

Comment:
don't need user_context here, but as a part of the mutator function. And then the user_context will get passed into the mutator function in the manage_session_post_response